bindings/python/scripts/context.py

```diff
@@ -0,0 +1,6 @@
+import os
```

Comment:
If you're branches are dependent on each other, change the base branch of the PR so the same diff doesn't appear in 2 PRs